scripts/question_answering/question_answering.py

```diff
@@ -0,0 +1,324 @@
+
+        self._batch_size = batch_size
+
+        with self.name_scope():
+            self._modeling_layer = LSTM(hidden_size=input_dim, num_layers=nlayers, dropout=dropout,
+                                        bidirectional=biflag, input_size=input_size)
+
+    def hybrid_forward(self, F, x, *args):
+        # pylint: disable=arguments-differ,unused-argument,missing-docstring
+        out = self._modeling_layer(x)
+        return out
+
+
+class BiDAFOutputLayer(HybridBlock):
+    """
+    ``BiDAFOutputLayer`` produces the final prediction of an answer. The output is a tuple of
+    start and end index of token in the paragraph per each batch.
+
+    It accepts 2 inputs:
+        `x` : the output of Attention layer of shape:
+        seq_max_length x batch_size x 8 * span_start_input_dim
+
+        `m` : the output of Modeling layer of shape:
+         seq_max_length x batch_size x 2 * span_start_input_dim
+
+    Parameters
+    ----------
+    batch_size : `int`
+        Size of a batch
+    span_start_input_dim : `int`, default 100
+        The number of features in the hidden state h of LSTM
+    nlayers : `int`, default 1
+        Number of recurrent layers.
+    biflag: `bool`, default True
+        If `True`, becomes a bidirectional RNN.
+    dropout: `float`, default 0
+        If non-zero, introduces a dropout layer on the outputs of each
+        RNN layer except the last layer.
+    prefix : `str` or None
+        Prefix of this `Block`.
+    params : `ParameterDict` or `None`
+        Shared Parameters for this `Block`.
+    """
+    def __init__(self, batch_size, span_start_input_dim=100, nlayers=1, biflag=True,
+                 dropout=0.2, prefix=None, params=None):
+        super(BiDAFOutputLayer, self).__init__(prefix=prefix, params=params)
+
+        self._batch_size = batch_size
+
+        with self.name_scope():
+            self._dropout = nn.Dropout(rate=dropout)
+            self._start_index_combined = nn.Dense(units=1, in_units=8 * span_start_input_dim,
+                                                  flatten=False)
+            self._start_index_model = nn.Dense(units=1, in_units=2 * span_start_input_dim,
+                                               flatten=False)
+            self._end_index_lstm = LSTM(hidden_size=span_start_input_dim,
+                                        num_layers=nlayers, dropout=dropout, bidirectional=biflag,
+                                        input_size=2 * span_start_input_dim)
+            self._end_index_combined = nn.Dense(units=1, in_units=8 * span_start_input_dim,
+                                                flatten=False)
+            self._end_index_model = nn.Dense(units=1, in_units=2 * span_start_input_dim,
+                                             flatten=False)
+
+    def hybrid_forward(self, F, x, m, mask, *args):
+        # pylint: disable=arguments-differ,unused-argument,missing-docstring
```

Comment:
Why `unused-argument` must be disabled?

Reply:
Same question for the other uses in this script

Reply:
I think it was because of *args in `hybrid_forward`. Removed it everywhere